webapp/src/js/controllers/messages-content.js

```diff
@@ -115,9 +115,35 @@ angular.module('inboxControllers').controller('MessagesContentCtrl',
         getContactable(id, type),
         MessageContacts.conversation(id)
       ])
+        .catch(function(err) {
+          if (err.code === 404) {
+            return $q.all([Promise.resolve(null), MessageContacts.conversation(id)]);
```

Comment:
You almost certainly don't need `null` here due to [JavaScript's use of falsy](https://developer.mozilla.org/en-US/docs/Glossary/Falsy).

If you **do** need `null` here for some esoteric reason please document it :-)